composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "c2sp/wycheproof",
+    "description": "Wycheproof test vectors for cryptographic software",
+    "authors": [
+        {
+            "name": "Scott Arciszewski",
+            "email": "scott.arciszewski@trailofbits.com",
+            "homepage": "https://github.com/tob-scott-a"
+        }
+    ],
+    "type": "library",
+    "keywords": ["wycheproof", "testing", "cryptography", "crypto"],
+    "license": "Apache-2.0",
+    "autoload": {
+        "psr-4": {
+            "C2sp\\Wycheproof\\": "php"
+        }
+    },
```

Comment:
@tob-scott-a just before I merge, should this part of the config be deleted to reflect 34982b8ac8f19cca05405bb7a4ba2b837f2bf117 ? 

(Also if you wouldn't mind adding a trailing newline to this file that would be great)